src/pages/help-centre/index.tsx

```diff
@@ -197,14 +197,20 @@ const HelpCentre = () => {
         setData({ ...data, search: sanitize(e.target.value) })
     }
 
-    const filtered_articles = matchSorter(data.all_articles, data.search.trim(), {
-        keys: ['title', 'sub_category'],
-    })
-
     const splitted_articles = is_eu_country
         ? euArticles(splitArticles(articles, 3))
         : splitArticles(articles, 3)
 
+    const articles_by_domain = is_eu_country
+        ? data.all_articles
+              .filter((el) => el.category !== eu_discards[0])
+              .filter((el) => el.category !== eu_discards[1])
+        : data.all_articles
+
```

Comment:
Please loop in to `eu_discards`. This can be done in one `filter` call.

Reply:
I forgot to push last changes .___. Done